### DISCUSSION TITLE
extend json and npz export

### DISCUSSION THREAD

**Discussion by j042** [tataratat/splinepy#257] [General]
- extend json export to support multipatches
- support `spline_data` export
- support `show_options` export

**Comment by jzwar** [comment]
Yes, that sounds like a good extension. Also HiWi-able?

**Comment by clemens-fricke** [comment]
Show options are now in #358 for the json ex- and import. Should we also add it to npz? What about spline data, is that still an issue?

**Reply by j042** [reply]
having it in npz export and spline data support would be great
